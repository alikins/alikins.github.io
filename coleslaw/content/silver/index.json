{
    "available_versions": {
        "v3": "v3/"
    },
    "_node_type": "IndexJsonNode",
    "name": "index.json"
}