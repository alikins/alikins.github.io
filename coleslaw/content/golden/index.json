{
    "available_versions": {
        "v3": "v3/"
    },
    "name": "index.json",
    "_node_type": "IndexJsonNode"
}